connector/src/main/java/org/jboss/as/connector/services/resourceadapters/deployment/ResourceAdapterDeploymentService.java

```diff
@@ -182,8 +201,8 @@ public CommonDeployment getRaDeployment() {
     }
 
     public AS7MetadataRepository getMdr() {
-            return mdr.getValue();
-        }
+        return mdr.getValue();
+    }
 
     private class AS7RaDeployer extends AbstractAS7RaDeployer {
```

Comment:
not really important, but we could probably rename this to WildFlyRaDeployer & AbstractWildFlyRaDeployer